### PR TITLE
FIX updated_at in addTimestamps method

### PR DESCRIPTION
**Pull request by DvDMH**

### CODE REVIEW

File: src/Phinx/Db/Table.php [hunk 1]
```diff
@@ -550,7 +550,8 @@ public function addTimestamps($createdAt = 'created_at', $updatedAt = 'updated_a
              ])
              ->addColumn($updatedAt, 'timestamp', [
                  'null' => true,
-                 'default' => null,
+	             'default' => 'CURRENT_TIMESTAMP',
```
Comment (lorenzo) [inline src/Phinx/Db/Table.php]: I think that for backwards compatibility, it should not have a default.

**Comment by rquadling** [comment]
Is there a way to get the fractional seconds in here?

**Comment by dereuromark** [comment]
ping @DvDMH

**Comment by dereuromark** [comment]
Seems to be a duplicate of https://github.com/cakephp/phinx/pull/1499/files
Either way we proceed (this or the other), they probably need a small test case.

**Comment by othercorey** [comment]
Closing due to no activity.

**Pull request closed without merging**